dump1090-fa/Dockerfile.template

```diff
@@ -14,7 +14,7 @@ ENV DUMP1090_ADAPTIVE_MAX_GAIN=""
 ENV DUMP1090_SLOW_CPU=""
 ENV REBOOT_DEVICE_ON_SERVICE_EXIT=""
 
-ARG PERM_INSTALL="tini lighttpd gettext-base libusb-1.0-0 libbladerf2 libhackrf0 liblimesuite22.09-1 librtlsdr0 rtl-sdr libsoapysdr0.8 libncurses6 libboost-system-dev libboost-program-options-dev libboost-regex-dev"
+ARG PERM_INSTALL="tini lighttpd gettext-base libusb-1.0-0 libbladerf2 libhackrf0 liblimesuite22.09-1 libsoapysdr0.8 libncurses6 libboost-system-dev libboost-program-options-dev libboost-regex-dev"
```

Comment:
It has `libsoapysdr0.8` here but then it's not installed in the same way as dump978 further down?

Not sure if it needs adding or whether the package is maybe not needed?

Reply:
Thanks for your feedback, @shawaj! I tried to build `libsoapysdr` from source, too, but that turned out to be very difficult. `soapysdr0.7` and `soapysdr-module-rtlsdr` depend on it, however, so that's why I'm installing it permanently early in the Dockerfile.

Reply:
@ketilmo my point on this was that you install libsoapysdr0.8 here, but then in the dump978 container you install some other stuff:

```
RUN	git clone https://github.com/pothosware/SoapyRTLSDR.git && \
	cd SoapyRTLSDR && \
	git checkout ${SOAPYRTLSDR_VERSION} && \
	mkdir build && cd build && cmake .. && make && make install && ldconfig && cd .. && \
	dpkg-buildpackage -b && cd .. && \
	dpkg -i soapysdr0.7*.deb soapysdr-module-rtlsdr_*.deb && \
	apt-mark hold soapysdr-module-rtlsdr
```

but in this container, you don't install that stuff. So I was wondering if there was a mistake in one or other of the containers

Reply:
@ketilmo I think we missed this before it got merged. Is this something we should fix?